src/coord/config.c

```diff
@@ -157,6 +170,11 @@ static RSConfigOptions clusterOptions_g = {
                          "Default is 30000 (30 seconds). 0 means no timeout.",
              .setValue = setTopologyValidationTimeout,
              .getValue = getTopologyValidationTimeout,},
+             {.name = "OSS_ACL_USERNAME",
```

Comment:
Can we do it only in OSS? (use `#ifdef RS_CLUSTER_ENTERPRISE`)

Reply:
We didn't do this until now, we would need more approvals to start doing it